shell/platform/tizen/external_texture.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_EXTERNAL_TEXTURE_H_
+#define EMBEDDER_EXTERNAL_TEXTURE_H_
+
+#include <atomic>
+#include <memory>
+#include "flutter/shell/platform/common/cpp/public/flutter_texture_registrar.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+
+#ifdef TIZEN_RENDERER_EVAS_GL
+#undef EFL_BETA_API_SUPPORT
+#include <Ecore.h>
+#include <Elementary.h>
+#include <Evas_GL_GLES3_Helpers.h>
+extern Evas_GL* g_evas_gl;
+EVAS_GL_GLOBAL_GLES3_DECLARE();
+#else
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+#include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
+#include <GLES3/gl32.h>
+#endif
+
+struct ExternalTextureGLState {
+  GLuint gl_texture;
+};
+
+static std::atomic_long kNextTextureId = {1};
```

Comment:
A leading "k" followed by mixed case is for constants only.
```suggestion
static std::atomic_long next_texture_id = {1};
```

Reply:
Thanks for your suggestion. I have changed to next_texture_id